doc/documentation_workflow.rst

```diff
@@ -0,0 +1,152 @@
+We use **Sphinx** to generate documentation and **Read the Docs** to publish it.
+
+The documentation for NEST is contained in the ``docs`` directory within the `NEST source code repository <https://github.com/nest/nest-simulator>`_ on GitHub.
+
+Using **GitHub** allows the NEST simulator documentation to live alongside its code. Our developers benefit from smooth version control, with each release of NEST having its own documentation.
+
+.. image:: _static/img/documentation_workflow.png
+  :width: 300
+  :alt: Alternative text
+
+Changing the documentation
+++++++++++++++++++++++++++
+
+If you notice something out of place, you can submit an `issue <https://github.com/nest/nest-simulator/issues>`_ in our GitHub repository.
+
+You can also make changes directly to your forked copy of the `NEST source code repository <https://github.com/nest/nest-simulator>`_ and create a `pull request <https://github.com/nest/nest-simulator/pulls>`_. Just follow the workflow below!
+
+Setting up your environment
++++++++++++++++++++++++++++
+
+To keep things simple, we have created a conda environment for you. Installing it will enable you to smoothly generate documentation for NEST.
+
+If you are using Linux and want to install a full development environment:
+
+1. Install conda (we recommend `miniconda <https://docs.conda.io/en/latest/miniconda.html#>`_).
+
+2. Switch to the ``doc`` folder in the source directory:
+
+.. code-block:: bash
+
+    cd git/nest-simulator/doc
+
+3. Create and activate the environment:
+
+.. code-block:: bash
+
+   conda env create -f conda_environment.yml
+   conda update -n base -c defaults conda
+   conda activate doc
+
+4. If you want to deactivate or delete the build environment:
+
+.. code-block:: bash
+
+   conda deactivate
+   conda remove --name doc --all
+
+Generating with Sphinx
+++++++++++++++++++++++
+
+Now that you activated your environment, you can generate HTML files using `Sphinx <https://www.sphinx-doc.org/en/master/>`_.
+
+Sphinx uses reStructuredText. To learn more on how to use it, check out this `quick reference <https://docutils.sourceforge.io/docs/user/rst/quickref.html>`_.
+
+Editing and creating pages
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. You can edit and/or add ``.rst`` files in the ``doc`` directory using your editor of choice.
+
+2. If you create a new page, open ``index.rst`` in the ``doc`` directory and add the file name under ``.. toctree::``. This will ensure it appears on the NEST simulator documentation's table of contents.
+
+3. Save your changes.
+
+Rendering HTML
+~~~~~~~~~~~~~~
+
+Using Sphinx, you can build documentation locally and preview it offline:
+
+1. Go to the ``doc`` folder in the source directory:
+
+.. code-block:: bash
+
+    cd </path/to/nest_source>/doc
+
+2. Generate HTML files:
+
+.. code-block:: bash
+
+   make html
+
+3. Preview files. They are then located in ``./docs/_build/html``:
+
+.. code-block:: bash
+
+   cd </path/to/nest_source>/doc/_build/html
+   firefox FILENAME.html
```

Comment:
Must it be firefox?

Reply:
No. It can be any browser.

Reply:
Changed ``firefox FILENAME.html`` to ``browser filename.html``